tests/Tests/600__ai_apps/680__starburst/680__starburst.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Library             SeleniumLibrary
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+
+Test Setup          Dashboard Test Setup
+Test Teardown       Dashboard Test Teardown
+
+
+*** Variables ***
+${SB_CARDS_XP}=     //article[@id="starburst"]
+${SB_BETA_DESC}=    //*[@class="pf-c-drawer__panel-main"]//div[@class="pf-c-alert pf-m-inline pf-m-info"]
+
+
+*** Test Cases ***
+Verify if the Starburst Beta text has been removed from Getting Started
+    [Tags]    ODS-1158    ODS-605
```

Comment:
This doesn't have any Tier tags and as such would never be run. Shouldn't we at least tag it with Tier2 or Sanity?

Reply:
it could be considered as Sanity